ios/FluentUI/Vnext/Avatar/Avatar.swift

```diff
@@ -6,8 +6,8 @@
 import UIKit
 import SwiftUI
 
-/// Properties available to customize the state of the avatar
-@objc public protocol MSFAvatarState {
+/// Properties available to customize the state of the Avatar in an Avatar Group
+@objc public protocol MSFAvatarGroupItemState {
```

Comment:
These protocols need to be defined within the scope of the AvatarGroup control.

Please take a look at the [MSFPersonaButtonAppearance](https://github.com/microsoft/fluentui-apple/blob/b992a3ceaca09501e34f86205512e212182a11df/ios/FluentUI/Vnext/PersonaButton/PersonaButton.swift#L15) and [MSFPersonaButtonData](https://github.com/microsoft/fluentui-apple/blob/b992a3ceaca09501e34f86205512e212182a11df/ios/FluentUI/Vnext/PersonaButton/PersonaButton.swift#L36) and how they're being used.